install_config/install/docker_registry.adoc

```diff
@@ -169,9 +170,68 @@ $ oadm registry --service-account=registry \
 ----
 endif::[]
 
-[[viewing-logs]]
 
+[[registry-ip-address]]
+=== Registry IP Address
+
+OpenShift refers to the integrated registry by its service IP address,
+so if you decide to delete and recreate the docker-registry service,
+you can ensure a completely transparent transition by arranging to
+re-use the old IP address in the new service.
+If a new IP address cannot be avoided, you can minimize cluster
+disruption by rebooting only the masters.
+
+[[re-using-the-address]]
+Re-using the Address::
+
+To re-use the IP address, you must save the IP address of the old docker-registry
+service prior to deleting it, and arrange to replace the newly assigned IP address
+with the saved one in the new docker-registry service.
+
+// NB: Snarfed from <https://github.com/openshift/openshift-docs/issues/1494>.
+. Make a note of the `ClusterIP` for the service.
```

Comment:
Some :'s after the step descriptions leading into the command:

"Make a note of the `ClusterIP` for the service:"